src/ORM/Interfaces/ValidatableInterface.php

```diff
@@ -5,6 +5,13 @@
 
 interface ValidatableInterface extends EntityInterface
 {
+    /**
+     * Get all properties for this entity
+     *
+     * @return string[]
+     */
+    public function getProperties(): array;
```

Comment:
This is also used in Entity when resolving a property, so maybe it needs to go on the magic entity interface as well?

Reply:
Resolved